libnmstate/schema.py

```diff
@@ -351,6 +351,16 @@ class LinkAggregation:
             class Slave:
                 NAME = "name"
 
+            class Options:
+                DOWN_DELAY = "bond-downdelay"
```

Comment:
Why do the options need the `bond-` prefix? Since they are already in the LinkAggregation subtree it should be already clear what they belong to.

Reply:
These correspond 1:1 with the OVS property names.
"Our" property name does not include it.

It will be simpler to keep 1:1 naming for simplicity at this stage although it is not a must.